### DISCUSSION TITLE
Welcome to gis_agm_series Discussions!

### DISCUSSION THREAD

**Discussion by ma-robles** [grupoioa/gis_agm_series#13] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MaCuur** [comment]
para qué es la flecha hacia arriba de los comentarios¿? jejeje

**Reply by ma-robles** [reply]
ups! aquí era la respuesta de la flecha!... creo que es para votar
